src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -221,8 +226,7 @@ public void Dispose()
                 disposed = true;
             }
         }
```

Comment:
nit: why is the blank line between the methods being deleted?  and while you're making changes here could you fix the indention on line #229.